packages/utilities/src/focus.ts

```diff
@@ -38,6 +38,19 @@ export function getLastFocusable(
   return getPreviousElement(rootElement, currentElement, true, false, true, includeElementsInFocusZones);
 }
 
+/**
+ * Gets the first tabbable element.
+ *
+ * @public
+ */
+export function getFirstTabbable(
+  rootElement: HTMLElement,
+  currentElement: HTMLElement,
+  includeElementsInFocusZones?: boolean): HTMLElement | null {
+
+  return getNextElement(rootElement, currentElement, true, false, false, includeElementsInFocusZones, false, true);
```

Comment:
Can you add inline comments for the boolean values to make it easier to see what this call is passing to getNextElement?

Reply:
sure.